test/integration/builder/selects.js

```diff
@@ -1,6 +1,7 @@
 /*global expect, d*/
 'use strict';
 
+const mysqlErrors = require('mysql/lib/protocol/constants/errors');
```

Comment:
Why depend on non-public part of external dependency and not just hardcode the constant internally?

Reply:
Emm, OK. I will change it

Reply:
My concern is that if there is a major change in the driver and both the constant is no longer there and field is no longer set, comparison is still going to be "undefined === undefined" and will pass. Thanks for changing it!